### PR TITLE
Prevent Balance null in swap. fix #203

### LINKED ISSUE #203: Swap page will crash when swap coin balance is null

### PR DESCRIPTION
**Pull request by imstar15**
image

### CODE REVIEW

File: src/common/translations/en.json [hunk 1]
```diff
@@ -444,6 +444,10 @@
       "body": "We've launched support for RNS, or RIF Name Service which enables the identification of blockchain addresses by human-readable names.",
       "confirmText": "Let’s try it",
       "cancelText": "skip"
+    },
+    "balanceInvalid": {
+      "title": "Balance Invalid",
+      "body": "Balance of this token is invalid. please try again later!"
```
Comment (chrisli30) [inline src/common/translations/en.json:450]: "title": "Wallet balance is not available.",
"body": "We are having problem retrieving your wallet balance. Please try again later."

File: src/common/translations/zh.json [hunk 2]
```diff
@@ -442,8 +442,8 @@
       "cancelText": "跳过"
     },
     "balanceInvalid": {
-      "title": "余额无效",
-      "body": "代币的余额无效，请重试!"
+      "title": "钱包余额无效",
```
Comment (chrisli30) [inline src/common/translations/zh.json:445]: 这是谷歌翻译的吗？不是无效吧

"title": "余额获取失败",

File: src/common/translations/es.json [hunk 3]
```diff
@@ -442,7 +442,7 @@
     },
     "balanceInvalid": {
       "title": "Saldo inválido",
```
Comment (chrisli30) [inline src/common/translations/es.json:444]: 这个title肯定变动过了，没翻译

File: src/common/translations/zh.json (hunk 2 above)
Comment (imstar15) [reply src/common/translations/zh.json]: 已修改

File: src/common/translations/es.json (hunk 3 above)
Comment (imstar15) [reply src/common/translations/es.json]: 已修改

**Pull request merged**